integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/management/CarbonAddLoadCommand.scala

```diff
@@ -73,15 +74,19 @@ case class CarbonAddLoadCommand(
       throw new MalformedCarbonCommandException("Unsupported operation on non transactional table")
     }
 
+    if (carbonTable.isChildTable) {
```

Comment:
```suggestion
    if (carbonTable.isChildTable) {
    if (carbonTable.isChildTable || carbonTable.isChildDataMap) {
```
ischildtable check is for MV datamap only

Reply:
ok